### PR TITLE
Add flag to optionally write HETATM tags in PDBs

### PR DESCRIPTION
**Pull request by kristofarkas**
HETATM tags in PDB can cause problems for certain use cases. For example NAMD doesn't recognise them and raises errors. Adding an option to `write_pdb` to optionally use `ATOM  ` instead of `HETATM` fixes this problem. The default is still the same as was before.

### CODE REVIEW

**Comment by swails** [comment]
Can one of the admins verify this patch?

**Comment by kristofarkas** [comment]
@swails added a test

File: test/test_parmed_formats.py [hunk 1]
```diff
@@ -1014,6 +1014,28 @@ def test_pdb_write_standard_names_water(self):
...
+        a = Atom(name='CA', atomic_number=6)
+        structure.add_atom(a, 'ASH', 2, 'A')
+        structure.add_atom(a, 'DG', 2, 'A')
+        structure.add_atom(a, 'T', 2, 'A')
+        structure.add_atom(a, 'MOL', 2, 'A')
+
+        coordinates = np.zeros((len(structure.atoms), 3))
+
+        structure.write_pdb("hetatom.pdb", use_hetatoms=True, coordinates=coordinates)
+        structure.write_pdb("no_hetatom.pdb", use_hetatoms=False, coordinates=coordinates)
```
Comment (hainm) [inline test/test_parmed_formats.py]: you can write to StringIO to avoid writing to disk.

**Comment by kristofarkas** [comment]
@hainm @swails any other comments?

**Comment by swails** [comment]
Can you update the or with the latest master branch?

**Comment by swails** [comment]
After that one final comment, this is good to go and I'll merge.

Thanks for bearing with me!

**Comment by kristofarkas** [comment]
@swails requested changes made!

**Pull request merged**